src/generator_http.rs

```diff
@@ -97,7 +97,7 @@ struct DbRequest {
 
 // implement a helper to get the http request which corresponds to the db request structure above
 impl DbRequest {
```

Comment:
async this code, same as the worker. Small refactor also, this was a little ugly